_data/people.yml

```diff
@@ -238,6 +236,12 @@
     role: MS
     employer: 
     employer_url: 
+  - name: Erik Nygaard
+    url: 
+    year: 2011
+    Role: MS
```

Comment:
`role` is not appearing when built. I think it needs to be lowercase.
![nygaard](https://user-images.githubusercontent.com/4060017/123272649-8e8b3400-d4c7-11eb-9c3b-ee7b899da4ce.png)